watertap/core/membrane_channel_base.py

```diff
@@ -303,19 +330,19 @@ class FrictionFactor(Enum):
 CONFIG_Template.declare(
     "friction_factor",
     ConfigValue(
-        default=FrictionFactor.flat_sheet,
+        default=FrictionFactor.default_by_module_type,
         domain=In(FrictionFactor),
         description="Darcy friction factor correlation",
         doc="""
         Options to account for friction factor correlations.
 
-        **default** - ``FrictionFactor.flat_sheet`` 
+        **default** - ``FrictionFactor.default_by_module_type`` 
 
     .. csv-table::
         :header: "Configuration Options", "Description"
 
-        "``FrictionFactor.flat_sheet``", "Friction factor correlation for flat-sheet membrane modules"
-        "``FrictionFactor.spiral_wound``", "Friction factor correlation for spiral-wound membranes"
+        "``FrictionFactor.default_by_module_type``", "Friction factor correlation that is specific to the supported membrane modules type"
+        "``FrictionFactor.custom``", "Custom user-defined friction factor correlations"
```

Comment:
```suggestion
        "``FrictionFactor.custom``", "Custom user-defined friction factor correlations"
```
I suppose we should delete for the time being.

Reply:
Done!